src/project/manifest/mod.rs

```diff
@@ -1022,7 +1033,7 @@ impl<'de> Deserialize<'de> for ProjectManifest {
                 name: EnvironmentName::Default,
                 features: Vec::new(),
                 features_source_loc: None,
-                solve_group: None,
+                solve_group: Some(solve_groups.insert("default", 0)),
```

Comment:
It will only create a "default" solve-group if the default environment is not redefined. Should we make sure it is created even if the default environment is redefined?